python/example_load_gen.py

```diff
@@ -20,17 +24,19 @@ def initialize_logging(level: int) -> None:
     root_logger.setLevel(level)
 
     handler = colorlog.StreamHandler()
-    handler.setFormatter(colorlog.ColoredFormatter(
-        "%(asctime)s %(log_color)s%(levelname)-8s%(reset)s %(thin_cyan)s%(name)s%(reset)s %(message)s"
-    ))
+    handler.setFormatter(
+        colorlog.ColoredFormatter(
+            "%(asctime)s %(log_color)s%(levelname)-8s%(reset)s %(thin_cyan)s%(name)s%(reset)s %(message)s"
+        )
+    )
     handler.setLevel(level)
     root_logger.addHandler(handler)
 
 
 class AsyncSetGetResult(Enum):
-    SUCCESS = 'SUCCESS',
-    UNAVAILABLE = 'UNAVAILABLE',
-    DEADLINE_EXCEEDED = 'DEADLINE_EXCEEDED',
+    SUCCESS = ("SUCCESS",)
```

Comment:
:whoa: this is new to me, is this the idiomatic way to define enums in Python?

Reply:
This change is courtesy of flake8. It's basically just turning an implicit tuple into an explicit one.